### ISSUE TITLE
4lang issues

### ISSUE THREAD

**Issue by GKingA** [recski/tuw-nlp#30]
There are some cases, where half of the sentence is left out right after a conjunction.
For example:
 > Meine Mutter hat mir erzählt, dass mein Vater einen Wahlkreiskandidaten nicht gewählt hat, *weil der gegen die Homo-Ehe ist ☺*

 > @HugoMuellerVogg @spdde Hat Rente nichts *mit sozialer Gerechtigkeit zu tun?*

Sometimes only the first word is kept from the sentence:

> Verloren *wäre die Zeit, in der wir nicht als Menschen gelebt, Erfahrungen gemacht, gelernt,geschaffen, genossen und gelitten hätten.*

Twitter specific issue:
 > @JoeHellBack @ThomasMichael71 @alexa_jung60 @Tschonka @dasstimmvieh @Mohrenpost Och, antifeministisch bin ich auch :)

becomes

 > (u_1 / ATSYMBOLJoeHellBack)

and

 > #Lesetipp: Rede von Prof. Dr. Friedhelm Hengsbach SJ: »@Umfairteilen jetzt!« #Sozialwort #Vermögenssteuer #Kapitalismus (via @WiuGe)

becomes

 > (u_1 / HASHTAGLesetipp)	(u_1 / HASHTAGSozialwort)

**Comment by recski** [comment]
fixed in https://github.com/recski/tuw-nlp/commit/87080ea46afd488d3a7d2f08568b146137debfec

**Comment by Eszti** [comment]
Can this issue be closed?

**Comment by recski** [comment]
yes, I think so, the 4lang issues mentioned here were fixed and the twitter-specific things are taken care of in the preprocessing, I think. But @GKingA can verify - and she may have run into new German-specific issues recently.
